sdks/typescript/CHANGELOG.md

```
@@ -1,5 +1,16 @@
 # Changelog
 
+## [2.0.0](https://github.com/langwatch/langwatch/compare/typescript-sdk@v1.2.0...typescript-sdk@v2.0.0) (2026-08-07)
+
+
+### ⚠ BREAKING CHANGES
+
+* **evaluators:** evaluations, monitors and experiments referencing a legacy/ragas_* evaluator type stop working. Their current equivalents in the ragas/* family remain available.
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)